server/rest/src/test/java/org/infinispan/rest/RestOperationsTest.java

```diff
@@ -37,10 +37,14 @@
    private HttpClient client;
```

Comment:
If you agree with my previous comment, changes in this file can be reverted.

Reply:
I'd rather keep the changes. It does not make a difference in test execution time.